core/templates/dev/head/filters.js

```diff
@@ -196,8 +196,28 @@ oppia.filter('parameterizeRuleDescription', [
           } else {
             // The following case is for MultipleChoiceInput
             for (var i = 0; i < choices.length; i++) {
+              var stringChange = true;
+              var choicesLabelString = choices[i].label;
+              while(stringChange) {
+                var nonInteractiveInputs = ['collapsible', 'image', 'link',
+                                            'math', 'tabs', 'video'];
+                for (var j = 0; j < nonInteractiveInputs.length; j++) {
+                  if (choicesLabelString.includes('<oppia-noninteractive-' +
```

Comment:
Use `indexOf` instead of `includes` since the latter isn't compatible in all browsers.